src/QuickMarcEditor/QuickMarcEditorContainer.js

```diff
@@ -95,15 +95,11 @@ const QuickMarcEditorContainer = ({
         setInstance(instanceResponse);
         setMarcRecord(formattedMarcRecord);
         setLocations(locationsResponse);
+        setIsLoading(false);
       })
       .catch(() => {
-        setInstance();
-        setMarcRecord();
-
         showCallout({ messageId: 'ui-quick-marc.record.load.error', type: 'error' });
-      })
-      .finally(() => {
-        setIsLoading(false);
+        history.goBack();
```

Comment:
Let's use `closeEditor` function instead - it's going to redirect to Instance page regardless what the previous page was

Reply:
Thanks, fixed.